File: 0-count-performance/go/count.go
package main

import (
	"fmt"
	"time"
	// "github.com/bradhe/stopwatch"
)

func main() {
	count := 0
	start := time.Now()
	for i := 0; i < 1000000000; i++ {
		count += 1
	}
	stop := time.Now()
	elapsedTime := stop.Sub(start)
	fmt.Printf("Count is %v\n", count)
	fmt.Printf("Execution time %v seconds", elapsedTime.Seconds())
}

File: 0-count-performance/typescript/count.ts
let count = 0;
const start = new Date().getTime();
for (let i = 0; i < 1000000000; i++){
  count += 1;
}
const end = new Date().getTime();
const elapsedTime = (end - start) / 1000.0;
console.log(`Count is ${count}`);
console.log(`Execution time ${elapsedTime} seconds`);